pkg/client/helper.go

```diff
@@ -228,6 +230,12 @@ func TransportFor(config *Config) (http.RoundTripper, error) {
 		if tlsConfig != nil {
 			transport = &http.Transport{
 				TLSClientConfig: tlsConfig,
```

Comment:
These are the exact same parameters that are in http.DefaultTransport. If you just add the TLSClientConfig to the DefaultTransport will that cause it to be applied elsewhere that we want to use the DefaultTransport (and if so, then using DefaultTransport anywhere seems like a terrible idea since it's a global variable).